### PR TITLE
[Assignement-1] Docker

### PR DESCRIPTION
**Pull request by Touqeerjadoon**
Update docker-compose.yaml file.
Remove Dockerfile comments.
Fix bugs.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN mkdir -p /home/app
+
+COPY ./app /home/app
+
+WORKDIR /home/app
+
+RUN npm install
+
+CMD ["node", "server.js"]
```
Comment (Abdullahxz) [inline Dockerfile]: Dockerfile is still lacking best practices. Please have a look here
https://docs.docker.com/build/building/best-practices/

**Review by Abdullahxz — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ENV MONGO_DB_USERNAME=admin \
+    MONGO_DB_PWD=admin
+
+WORKDIR /home/app
+
+COPY ./app/package*.json ./
+
+RUN npm install --production
+
+COPY ./app /home/app
```
Comment (Abdullahxz) [inline Dockerfile]: I think you do not need to use `/home/app` with `WORKDIR` directive.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+EXPOSE 3000
```
Comment (Abdullahxz) [inline Dockerfile:14]: Is this needed?

**Review by Abdullahxz — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (Touqeerjadoon) [reply Dockerfile:14]: It's a optional step but it helps other developers understand the port configuration. If not needed, i can remove it.

File: Dockerfile (hunk 2 above)
Comment (Touqeerjadoon) [reply Dockerfile]: yes, you are right it already set in work directory.

Comment (Abdullahxz) [reply Dockerfile]: Please update this in the dockerfile.

**Review by Abdullahxz — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (Touqeerjadoon) [reply Dockerfile]: Updated, please review my Dockerfile again.

**Review by Abdullahxz — APPROVED** [review]
Nice one

**Pull request merged**